src/hooks/persisted/useSettings.ts

```diff
@@ -163,7 +163,7 @@ const initialAppSettings: AppSettings = {
 const initialBrowseSettings: BrowseSettings = {
   showMyAnimeList: true,
   showAniList: true,
-  globalSearchConcurrency: 1,
+  globalSearchConcurrency: 3,
```

Comment:
most phones should be able to handle 3 and the first few concurrency increases makes a massive difference in loading time so may as well increase default